packages/components/src/VirtualizedList/VirtualizedList.component.js

```diff
@@ -73,6 +74,7 @@ function VirtualizedList(props) {
 VirtualizedList.displayName = 'VirtualizedList';
 VirtualizedList.propTypes = propTypes;
 VirtualizedList.defaultProps = {
+	defaultHeight: 250,
```

Comment:
question: why 250 ?

Reply:
`$> ./randomValue.sh`, did you prefer another one?

Reply:
I don't care, I don't want `height: 0` :)